packages/rocket-static-sites-aws-infrastructure/.mocharc.yml

```diff
@@ -0,0 +1,10 @@
+diff: true
+require: 'ts-node/register'
+extension:
+  - ts
+package: './package.json'
+recursive: true
+reporter: 'spec'
+timeout: 5000
+full-trace: true
+bail: true
```

Comment:
Curious: Shouldn't this be inside the test folder and not in the root?

Reply:
It looks like we have this file in the package root in each of the packages 🤔 

I guess that mocha will look for it recursively, If you think we should move it, let's create an issue to do this in a separate PR in all packages!